script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_1_1.txt

```diff
@@ -37,7 +37,7 @@ Noel-sensei is not a strikingly beautiful woman.//ノエル先生は目を見張
 But it's like she's within our reach.//その分手が届く範囲の憧れというか、
 Or she has that kind of big sister aura.//ちょっとだけ年上のお姉さんというか、
 Or maybe she's filled with a charm that can't be put into words.//言葉にはできない魅力で満ちているというか、
-Anyways, that was something like a fluffy whip that stimulated the yearnings of high school boys at first sight.
+Anyway, that was something like a fluffy whip that instantly stimulated the yearnings of high school boys.
```

Comment:
>とにかく、一目で男子高校生の憧れを刺激する、ふわふわしたホイップのような何かがあった。
>All that to say, just like a fluffy whip[?] she possessed some quality that would stir up the desires of the high school boys the moment they laid eyes on her. 

Although is fluffy whip correct? It's ふわふわしたホイップ seems like it might mean whipped _cream_, not :flushed: whips. Are we over-reading this?